Femlives/gfnmjwft
# Feat/#66/csrf protection

## Description

- new npm packages: luxon & nanoid
- renames and refactors generateToken to generateAuthToken
- 
## How Can This Be Tested?

- start project
- try to do some csrf hacking :P
- check your cookies in the browser to see if the session signature cookie is set

## Checklist

- [x] 🙅‍♀️ No leftover console.logs
- [x] 🦾 Sufficient tests, e. g. unit test
- [x] 🧠 Left over TODOs are marked with a url to the ticket
